upup/models/cloudup/resources/addons/aws-cloud-controller.addons.k8s.io/k8s-1.18.yaml.template

```diff
@@ -1,17 +1,16 @@
 ---
 apiVersion: apps/v1
-kind: DaemonSet
+kind: Deployment
 metadata:
   name: aws-cloud-controller-manager
   namespace: kube-system
   labels:
     k8s-app: aws-cloud-controller-manager
 spec:
+  replicas: 1
```

Comment:
Wouldn't we want some redundancy for this, like we do for controller-manager?

Reply:
Yeah, maybe start with a default of 2.

Reply:
You can set repicas to 1 or 2 based on the amount of CP nodes here: `pkg/model/components/cloudconfiguration.go`
You'd just have to add instance groups to the struct. These are created in `upup/pkg/fi/cloudup/populate_cluster_spec.go`